dotnet8/sns/{{cookiecutter.project_name}}/src/SNSEventSource/aws-lambda-tools-defaults.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Information": [
+    "This file provides default values for the deployment wizard inside Visual Studio and the AWS Lambda commands added to the .NET Core CLI.",
+    "To learn more about the Lambda commands with the .NET Core CLI execute the following command at the command line in the project root directory.",
+    "dotnet lambda help",
+    "All the command line options for the Lambda command can be specified in this file."
+  ],
+  "profile": "",
+  "region": "",
+  "configuration": "Release",
+  "function-runtime": "dotnet8",
+  "function-memory-size": 256,
+  "function-timeout": 30,
```

Comment:
Just curiosity, how this is going to affect the lambda function which will be deployed? These are not used with CFN/SAM right?

Reply:
SAM CLI eventually calls into Amazon.Lambda.Tools which can read these, although I need to test that to be 100% sure.

Reply:
So any settings that affect `dotnet lambda package` will come into play with SAM CLI which can be seen with `dotnet lambda package --help`. Except what gets overridden in the CLI call. Like function-architecture here: https://github.com/aws/aws-lambda-builders/blob/57b74abb966c77a7c55546cc49d5fc9683cb6b0c/aws_lambda_builders/workflows/dotnet_clipackage/actions.py#L116